pkg/archive/copy.go

```diff
@@ -151,11 +154,19 @@ func CopyInfoSourcePath(path string, followLink bool) (CopyInfo, error) {
 		return CopyInfo{}, err
 	}
 
+	var dsize int64
+	if stat.Mode().IsDir() {
+		if dsize, err = directory.Size(resolvedPath); err != nil {
+			return CopyInfo{}, err
+		}
+	}
 	return CopyInfo{
 		Path:       resolvedPath,
 		Exists:     true,
 		IsDir:      stat.IsDir(),
 		RebaseName: rebaseName,
+		Size:       stat.Size(),
+		DSize:      dsize,
```

Comment:
But Size exists in "type ContainerPathStat struct" already, although we did not use Size here. And there are lots of other users of this struct. If the Size did not container its children before, I think that's possible to introduce regression.